### PR TITLE
feat(redirection): add jenkinsistheway.io redirection to stories.jenkins.io

### PR DESCRIPTION
**Pull request by smerle33**
as per : https://github.com/jenkins-infra/helpdesk/issues/2943
setting a redirection for jenkinsistheway.io to stories.jenkins.io

### CODE REVIEW

File: clusters/prodpublick8s.yaml [hunk 1]
```diff
@@ -253,3 +253,9 @@ releases:
       - "../config/rating.yaml"
     secrets:
       - "../secrets/config/rating/secrets.yaml"
+  - name: jenkinsisthewayio-redirect
+    namespace: jenkinsisthewayio
```
Comment (lemeurherve) [inline clusters/prodpublick8s.yaml:257]:
```suggestion
    namespace: jenkinsisthewayio-redirector
```

**Review by lemeurherve — APPROVED** [review]

File: clusters/prodpublick8s.yaml (hunk 1 above)
Comment (dduportal) [reply clusters/prodpublick8s.yaml:257]: the suffix was made to make it explicit that it does not host any application, only a redirection

Comment (lemeurherve) [reply clusters/prodpublick8s.yaml:257]: Which suffix? `io`?

Comment (dduportal) [reply clusters/prodpublick8s.yaml]: > Which suffix? io?

Oh sorry my brain inverted up the 2 lines. Nevermind my comment.

Comment (lemeurherve) [reply clusters/prodpublick8s.yaml]: I would even suggest "http-redirections" as namespace name, to regroup all redirections.

File: clusters/prodpublick8s.yaml [hunk 2]
```diff
@@ -253,3 +253,9 @@ releases:
       - "../config/rating.yaml"
     secrets:
       - "../secrets/config/rating/secrets.yaml"
+  - name: jenkinsisthewayio-redirect
+    namespace: jenkinsisthewayio-redirector
+    chart: jenkins-infra/httpredirector
+    version: 0.1.0
```
Comment (dduportal) [inline clusters/prodpublick8s.yaml:259]:
```suggestion
    version: 0.1.1
```

**Review by dduportal — APPROVED** [review]
🚀

**Pull request merged**